versioned/gc/iceberg-actions/pom.xml

```diff
@@ -30,11 +30,12 @@
 
   <name>Nessie - Versioned - GC for Spark - Iceberg - Actions</name>
 
-  <properties>
-    <iceberg.nessie.version>${project.version}</iceberg.nessie.version>
-  </properties>
-
   <dependencies>
+    <dependency>
+      <groupId>org.projectnessie</groupId>
+      <artifactId>nessie-client</artifactId>
+      <version>${iceberg.nessie.version}</version>
+    </dependency>
```

Comment:
Could be 
```

    <dependency>
      <groupId>org.projectnessie</groupId>
      <artifactId>nessie-client</artifactId>
      <version>${iceberg.nessie.version}</version>
      <scope>test</scope>
      <exclusions>
        <exclusion>
          <groupId>org.projectnessie</groupId>
          <artifactId>nessie-model</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
 ```

Reply:
context: we discussed this during standup where @snazy had a similar problem.

@snazy do you want to fix this as part of a follow-up PR or should I fix it here?

Reply:
I'll make a follow up PR shortly with a concrete problem description for this part